main_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func setup(t *testing.T) { //Create temp dir and config file
```

Comment:
This function is completely useless in that current form.

It only creates a file, then delete it before actually running the test.

Reply:
Oh okok I understood that wrong. Thanks !